src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css

```diff
@@ -3,12 +3,10 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.monaco-editor .accessibilityHelpWidget {
-	padding: 10px;
-	vertical-align: middle;
-	overflow: auto;
+.accessible-view {
+	position: relative;
 	background-color: var(--vscode-editorWidget-background);
 	color: var(--vscode-editorWidget-foreground);
 	box-shadow: 0 2px 8px var(--vscode-widget-shadow);
-	border: 2px solid var(--vscode-contrastBorder);
+	border: 2px solid var(--vscode-contrastActiveBorder);
```

Comment:
Do you want a focus border on :focus here instead? I don't think contrast active border is set for light/dark themes.

Reply:
It's not focused (the editor input is). I believe this behaves correctly - a low vision user will be using a HC theme where this will be applied.